doc/userguide/lua/libs/file.rst

```diff
@@ -0,0 +1,171 @@
+File
+####
+
+File information is exposed to Lua scripts with the ``suricata.file``
+library, for example::
+
+  local filelib = require("suricata.file")
+
+Setup
+*****
+
+If your purpose is to create a logging script, initialize the script
+as:
+
+::
+
+  function init (args)
+     local needs = {}
+     needs["type"] = "file"
+     return needs
+  end
+
+Currently the Lua file library is not implemented for rules.
```

Comment:
IFF there's a new version of this PR: maybe emphasize this, with a `Note` or bold?